groovy/replica-pipeline

```diff
@@ -0,0 +1,39 @@
+#!/bin/groovy
+
+def GIT_URL = "https://github.com/sartura/replica.git"
+
+pipeline {
+  agent {
+      label "centos7-docker-aws-1c-2g"
+  }
+  stages {
+    stage("Prepate Env"){
+      steps{
+        script {
+          echo "Clone Sources"
+          checkout(
+            changelog: false,
+            poll: false,
+            scm: [$class: 'GitSCM',
```

Comment:
Is it necessary to define this explicitly? This looks like a pretty standard checkout.

Reply:
In this case yes since I am not using pipeline SCM, but rather pipeline script. 
The groovy script will live in ci-management for now rather than the dev's repo